docs/configuration.md

```diff
@@ -0,0 +1,97 @@
+# Configuration
+
+## Persistency
+
+**Please note:** Per default configuration, the data saved in each service is not persisted and will get lost every time the service restarts. In order to make stored data, database and the search index persistent, the `volumes` lines in the docker-compose.yml file need to be uncommented. For example:
```

Comment:
According to the change in https://github.com/k-box/k-link/pull/3/commits/a420e6348971bf36feed3d9b1879ce86ea5cac28#diff-5076e32e884a50b3050daacee130db2aR90 this is not true, as all volumes configuration where uncommented